Gemfile

```diff
@@ -14,6 +14,10 @@ gem "redcarpet"
 gem "sentry-raven"
 gem "unicorn"
 
+gem "cssbundling-rails", "~> 1.2"
+gem "jsbundling-rails", "~> 1.1"
+gem "sprockets-rails", "~> 3.4"
+
```

Comment:
> I appear to need sassc-rails to run the specs

@pablobm It shouldn't be necessary. I'll double-check! The goal is to use the `sass` command from node package and not rely on the gem anymore